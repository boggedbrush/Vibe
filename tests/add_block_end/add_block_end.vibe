# VibeSpec: 1.2
patch_type: add_block
file: hello.py
position: end
--- code: |
  # >>> Inserted at EOF
  if __name__ == "__main__":
      greet("World")
