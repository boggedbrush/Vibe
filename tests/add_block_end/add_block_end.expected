# hello.py
def greet(name):
    print(f"Hello, {name}!")


class Greeter:
    def __init__(self, name):
        print(f'Hello {name}')


# >>> Inserted at EOF
if __name__ == "__main__":
    greet("World")
